deliver/lib/deliver/runner.rb

```diff
@@ -55,7 +55,7 @@ def run
       verify_version if options[:app_version].to_s.length > 0 && !options[:skip_app_version_update]
 
       # Rejecting before upload meta
-      # Screenshots can not be update/deleted if in waiting for review
+      # Screenshots cannot be updated or deleted if the app is in the "waiting for review" state
```

Comment:
fwiw, i have a rule (which I'm not using because it's too complicated) that generally complains about `not...or` because sometimes there should be a `neither`/`nor` pattern. But both (a) correctly applying a rule and (b) correctly describing the rule to catch incorrect cases are too hard, so I just cry a little bit when I see things like this. 

I do agree that expanding the `/` is the right thing as is fixing the tense agreement.

I suspect this could be `can be neither ... nor...while...`, but that's really too much and I'd rather not deal w/ it in this PR.

Reply:
I love your attention to detail, can we work together on more projects? 😆